digdag-standards/src/test/java/io/digdag/standards/operator/pg/PgConnectionTest.java

```diff
@@ -111,9 +116,10 @@ public void executeUpdate()
 
     @Test
     public void txHelperPrepare()
-            throws SQLException
+            throws SQLException, NotReadOnlyException
     {
         TransactionHelper txHelper = pgConnection.getStrictTransactionHelper(null, "__digdag_status", Duration.ofDays(1));
+        doThrow(DatabaseException.class).when(pgConnection).executeReadOnlyQuery(eq("SELECT count(*) FROM \"__digdag_status\""), any());
```

Comment:
It would be better if you add an explanation of this mocking. IIUC, this mocking is to simulate the case when `__digdag_status` table does not exist. I think that case is a bit high context, so an explanation would be nice.

Reply:
Good point! And I'll add another case that status table doesn't exist in this test class as well as PgIT.